java/src/edu/wpi/first/wpilibj/networktables/NetworkTable.java

```diff
@@ -204,9 +204,11 @@ public synchronized static NetworkTable getTable(String key) {
   }
 
   private final String path;
+  private final String pathWithSep;
 
   NetworkTable(String path) {
     this.path = path;
+    this.pathWithSep = path + PATH_SEPARATOR;
```

Comment:
Yes there are some places that still need the raw path variable without a postpended separator (Table listeners don't need the sep)